static/js/controllers/reports-content.js

```diff
@@ -45,10 +47,34 @@
         setMessageState(group, 'muted', 'scheduled');
       };
 
+      $scope.toggleExpand = function(selection) {
+        if ($scope.selectMode) {
+          selection.expanded = !selection.expanded;
+        }
+      };
+
+      $scope.deselect = function(selection, $event) {
+        if ($scope.selectMode) {
+          $event.stopPropagation();
+          for (var i = 0; i < $scope.selected.length; i++) {
+            if ($scope.selected[i].report._id === selection._id) {
+              $scope.selected.splice(i, 1);
+              $('#reports-list li[data-record-id="' + selection._id + '"] input[type="checkbox"]')
+                .prop('checked', false);
+              return;
+            }
+          }
+        }
+      };
+
       Changes({
         key: 'reports-content',
         filter: function(change) {
-          return $scope.selected && $scope.selected._id === change.id;
+          return $scope.selected &&
```

Comment:
It probably will never be null, but there _might_ be a race condition where the Changes event fires as the controller is destroying itself. Because the destroy handling code and this code are both event based I don't fully trust that the controller will clean up in a deterministic fashion.